### PR TITLE
[LOOP-5035] Pump event time zone sync

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-5035

### CODE REVIEW

**Review by Camji55 — APPROVED** [review]

File: TidepoolServiceKit/Extensions/PersistedPumpEvent.swift [hunk 1]
```diff
@@ -173,6 +175,33 @@ extension PersistedPumpEvent: IdentifiableDatum {
...
+        }
+        
+        let fromTime = formattedDateWithoutTimeZoneOffset(date, for: TimeZone(secondsFromGMT: fromSecondsFromGMT))
+        let toTime = formattedDateWithoutTimeZoneOffset(date, for: TimeZone(secondsFromGMT: toSecondsFromGMT))
+        var datum = TTimeChangeDeviceEventDatum(time: date,
+                                                from: TTimeChangeDeviceEventDatum.Info(time: fromTime),
+                                                to: TTimeChangeDeviceEventDatum.Info(time: toTime),
+                                                method: .manual)
+        let origin = datumOrigin(for: resolvedIdentifier(for: TStatusDeviceEventDatum.self), hostIdentifier: hostIdentifier, hostVersion: hostVersion)
+        datum = datum.adornWith(id: datumId(for: userId, type: TStatusDeviceEventDatum.self),
```
Comment (darinkrauss) [inline TidepoolServiceKit/Extensions/PersistedPumpEvent.swift]: The `TStatusDeviceEventDatum` needs to be replaced with `TTimeChangeDeviceEventDatum`, I believe.

**Review by darinkrauss — CHANGES_REQUESTED** [review]
Just one change...

**Review by ps2 — APPROVED** [review]
LGTM! Once Darin's request is completed.

File: TidepoolServiceKit/Extensions/PersistedPumpEvent.swift (hunk 1 above)
Comment (nhamming) [reply TidepoolServiceKit/Extensions/PersistedPumpEvent.swift]: good catch!

**Review by darinkrauss — APPROVED** [review]
LGTM!

**Pull request merged**